lib/UserInterface/textinput/Editor.cpp

```diff
@@ -449,6 +450,9 @@ namespace textinput {
         ProcessMove(kMoveEnd, R);
         std::vector<std::string> completions;
         TabCompletion* tc = fContext->GetCompletion();
+        Reader* reader = fContext->GetReaders()[0];
+        StreamReaderUnix* streamReader = (StreamReaderUnix*)(reader);
```

Comment:
Yes, pushed!
What do we do with the vector of Readers? Is the first one good enough?